SECURITY.md

```diff
@@ -1,7 +1,66 @@
 # Security Policy
-Intel is committed to rapidly addressing security vulnerabilities affecting our customers and providing clear guidance on the solution, 
-impact, severity and mitigation. 
+As an open-source project, we understand the importance of and responsibility
+for security. This Security policy outlines our guidelines and procedures for
+ensuring the highest level of Security and trust for our users who consume
+oneTBB.
 
-## Reporting a Vulnerability
-Please report any security vulnerabilities in this project 
-[utilizing the guidelines here](https://www.intel.com/content/www/us/en/security-center/vulnerability-handling-guidelines.html).
+## Supported Versions
+Security vulnerabilities can be fixed in the [latest version][1] version
```

Comment:
Minor suggestion: 
Security vulnerabilities are fixed in the [latest version][1] version...

Reply:
Changes applied. Thank you.